flac.md

```diff
@@ -443,6 +445,13 @@ Value | Picture type
 19    | Band or artist logotype
 20    | Publisher or studio logotype
 
+In case not a picture but an URI is contained in this block, the following points apply:
+
+- In case an URI does not include a protocol specifier, the URI is considered relative to the FLAC file being processed. Such URIs are retrieved with the same protocol as the FLAC file being processed, if the protocol has a concept of relative URIs.
+- Applications MUST obtain explicit user approval to retrieve images via remote protocols, or when retrieving a file that is not in the directory of the FLAC file being processed.
```

Comment:
Maybe to be moved in the security part?

Reply:
I think it would be nicer to leave it here, because in the Security considerations it would be missing context I think. Also, this is the only place where the protocol specifies an URL.

On the other hand, Vorbis comments can also contain links, and some applications could be using certain tags to link to extra content. In that case the consideration would be more general indeed.

Reply:
I've now done both, there's a more generic version added to the security considerations.